src/layouts/section/cp-k8s-api-reference.html

```diff
@@ -4,12 +4,10 @@
   <div class="col-md-10 col-md-offset-1">
     <h1 id="title">Control Plane Kubernetes API Reference</h1>
 
+    <p>At Giant Swarm, the _Control Plane_ is a Kubernetes cluster that manages allows to create and manage the Kubernetesa clusters running your workloads (we call them _Tenant Clusters_).</p>
```

Comment:
That needs a little love. I am not sure what you want to get across. 

```
is a Kubernetes cluster that manages allows to create and manage
```

Reply:
Done